README.md

```diff
@@ -63,6 +63,7 @@ client = User(user_id="user_id")
 
 # Get all apps
 apps = client.list_apps()
```

Comment:
can we rename it to `apps_generator_obj` or something else

Reply:
Thanks for the catch 👍